test/data/example_data.py

```diff
@@ -112,8 +112,6 @@ def sort_messages(messages):
     [
         Message(is_fd=True, data=range(64)),
         Message(is_fd=True, data=range(8)),
-        Message(is_fd=True, bitrate_switch=True, is_remote_frame=True),
```

Comment:
Why did you remove these?

Reply:
I explained it in the commit message: "Remove RTR tests for FD frames since they doesn't exist
and thus cannot be represented in the canutils log format.
See "CAN with Flexible Data-Rate, version 1.0, Bosch,
April 17th, 2012, pp.11": "The REMOTE TRANSMISSION REQUEST
(RTR) bit only exists in CAN format frames. [...] There are
no REMOTE FRAMES in CAN FD format." "

Reply:
Ah, nice catch! Sorry, I did not recognize the additional lines in the message. Looks fine.